scripts/release/api-analyzer.js

```diff
@@ -158,6 +144,38 @@ const analyze = (file, type) => {
   let output;
   const data = fs.readFileSync(file, { encoding: 'utf8' });
   const meta = wca.analyzeText(data);
+
+  meta.results.forEach(result => {
+    result.componentDefinitions.forEach(definition => {
+      const propCollection = {};
+      // WORKAROUND: Modify meta data of properties/attributes to make it fit with api reference tables of "elf-docs"
+      definition.declaration.members.forEach(member => {
+        let { propName, attrName, kind } = member;
+        // Convert default value of properties to theirs actual type
+        if(member.default === 'null') {
+          member.default = null;
+        }
+        else if(member.default === '[]') {
+          member.default = [];
+        }
+        // Merge attributes that defined by JSDOC to properties table
+        if(propName && !attrName) {
+          propCollection[propName] = member;
+        }
+        if(kind === 'attribute' && !propName && attrName) {
+          const attrCamelCase = attrName.replace(/-./g, attr => attr.length > 0 ? attr[1].toUpperCase() : '');
```

Comment:
Does this means that we only support one hyphen attribute ?

Reply:
No it doesn't. This Regex support multiple hyphen.

Reply:
I see. Thanks.